src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -160,13 +160,15 @@ public function getConfigTreeBuilder()
 
         $this->addFormatSection($rootNode, 'formats', [
             'jsonld' => ['mime_types' => ['application/ld+json']],
-            'json' => ['mime_types' => ['application/json']], // Swagger support
             'html' => ['mime_types' => ['text/html']], // Swagger UI support
         ]);
         $this->addFormatSection($rootNode, 'error_formats', [
             'jsonproblem' => ['mime_types' => ['application/problem+json']],
             'jsonld' => ['mime_types' => ['application/ld+json']],
         ]);
+        $this->addFormatSection($rootNode, 'doc_formats', [
```

Comment:
Can you rename this key, and all related variables `docs_format` (with a final s)? Thanks!